lib/db_ido_mysql/schema/upgrade/2.13.0.sql

```diff
@@ -0,0 +1,23 @@
+-- -----------------------------------------
+-- upgrade path for Icinga 2.13.0
+--
+-- -----------------------------------------
+-- Icinga 2 | (c) 2021 Icinga GmbH | GPLv2+
+--
+-- Please check https://docs.icinga.com for upgrading information!
+-- -----------------------------------------
+
+SET SQL_MODE="NO_AUTO_VALUE_ON_ZERO";
+
+-- ----------------------------------------
+-- #7472 Support hosts with >128 characters
+-- ----------------------------------------
+
+ALTER TABLE icinga_objects
+  MODIFY COLUMN name1 varchar(255) character set latin1 collate latin1_general_cs default '',
+  MODIFY COLUMN name2 varchar(255) character set latin1 collate latin1_general_cs default NULL;
+
+-- ------------------------------
+-- set dbversion (same as 2.11.0)
+-- ------------------------------
+INSERT INTO icinga_dbversion (name, version, create_time, modify_time) VALUES ('idoutils', '1.15.0', NOW(), NOW()) ON DUPLICATE KEY UPDATE version='1.15.0', modify_time=NOW();
```

Comment:
I'm not sure about the semantics here, but
1. shouldn't this be the same version as in `schema/mysql.sql`? (Looks like bumping in there was forgotten with the 2.11.0 schema upgrade)
2. why not make this `1.15.1`?

Reply:
1. Other construction area (which we should address nevertheless).
2. Have a look at 2.8 -> 2.8.1.

Reply:
1. -> #8852
2. Ok, then I don't get the point of this schema version but fine.